### ISSUE TITLE
notify / opt out when added to a starter pack or list

### ISSUE THREAD

**Issue by PennyBlake1827** [bluesky-social/social-app#5905]
### Describe the Feature

Starter packs and lists should have the option to opt out - either to never be added to them or to be only added by accounts you follow, or to opt out of individual packs you have been added to - and users should be notified every time when they're added to a list or pack. This is especially important as people are making negative / prejudiced / hate-based starter packs like 'people to block' or 'accounts not to follow' which isn't safe or inclusive. 

### Additional Context

 This is especially important as people are making negative / prejudiced / hate-based starter packs like 'people to block' or 'accounts not to follow' which isn't safe or inclusive.

**Comment by tanho63** [comment]
related to #5066 and #5493

**Comment by Leseratte10** [comment]
I agree that a user should be able to prevent being added to a starter pack. I don't think opt-in makes sense here since it's probably going to make starter packs basically useless if only people who familiarize themselves with what a starter pack is can be on one, but an opt-out to prevent being added to a starter pack, or a "ask me for permission every time I'm added to a starter pack" is probably a good idea. 

As for opt-out or permission to be added to a list - not a good idea. One huge advantage BlueSky has over X is that if there's a certain group of people you don't want to hear about, you can add a list of people to your block or mute list. If I don't want to hear about Trump, I can mute a list of people who constantly post about Trump. If I don't want to hear about Biden I can mute a list of people who constantly post about Biden. And so on. The whole point of custom labelers and custom moderation lists is for people to be able to customize their experience to not include content they don't want to see in their feed. 

It would just be like Twitter where Elon Musk decided that you can no longer block his account and have to see his posts in your feed even if you don't want to ...
